mock/datasets/no2.data.mdx

```diff
@@ -5,8 +5,12 @@ featuredOn:
   - covid-19
 description: "Since the outbreak of the novel coronavirus, atmospheric concentrations of nitrogen dioxide have changed by as much as 60% in some regions."
 usage:
-  url: 'https://nasa-impact.github.io/veda-documentation/timeseries-stac-api.html'
-  title: 'Time series using STAC API statistics endpoints'
+  - url: 'https://nasa-impact.github.io/veda-documentation/timeseries-stac-api.html'
+    description: Static notebook
```

Comment:
No 100% sold on the `description` property. Normally it denotes something longer and it is used as such in other parts of the app. I'd say this is more of a `label` perhaps?

Reply:
Done!